README.md

```diff
@@ -73,6 +73,8 @@ Unlike the web interface, you can specify the number of search results you would
 
 Note: v1.1 and below require the Python3 `requests` library to make HTTPS requests. This dependency is removed in the later releases.
 
+For clipboard copy support, install [`pyperclip`](https://pypi.python.org/pypi/pyperclip) from `pip`.
```

Comment:
We can add a little more clarity here:

For copy url to clipboard (at the omniprompt), install [`pyperclip`](https://pypi.python.org/pypi/pyperclip). This is an optional dependency.

Note: We can _omit_ the mention of `pip`. There may be several ways to install a Python module.